version.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Any reason we don't simply read this info from `package.json` instead of introducing  a new file which rewrites it?

Reply:
we decided to do it because exposing package.json to the client will send all dependency version numbers to the client. https://stackoverflow.com/questions/9153571/is-there-a-way-to-get-version-from-package-json-in-nodejs-code

Reply:
That information is still available as the code is open source. Just take the version, look at our GH repo for that versions and see the package.json.